### DISCUSSION TITLE
Difficulty selecting buttons and switches in UI

### DISCUSSION THREAD

**Discussion by njakeman** [alicevision/Meshroom#2084] [General]
I'm sure it can't be just me, but I'm experiencing difficulty selecting some of the sliders, buttons, and check options.
Mouse hover almost always shows the symbol for expanding the borders of a window/region.
If I hover over the 3 buttons in the top right of the graph editor, I cannot select any of them.
I just about managed to adjust thumbnail size in the image window, but that  seemed to require pixel level precision with the mouse.

Neither can I see the 'Load Model' button which I think is supposed tobe in the 3D viewer panel

Windows 11, Meshroom 2023, Screen res 3840 x 2160

**Comment by natowi** [comment]
This could be an issue with QT similar to https://github.com/qgis/QGIS/issues/40274
Check if you use a scaling factor in your Windows display settings. You can also adjust the settings software specific in the properties of the executable.
Could you share a screenshot of the actual representation?

**Comment by roydenyates** [comment]
I have this issue as well, on linux. It is difficult to capture as my screen capture shows the pointer as an arrow and not how it appears on monitor. Where an arrow pointer shows here,  on monitor it  is the double ended arrow for changing the size of the viewport. This triggers across about a centimetre of screen and so masks the the three vertical dots for the options.  One must search for the sweet spot, which is very small in order to gain access to the options. This issue occurs throughout with other controls - the trigger area for size change is too big it seems, at least on some systems. Fiddling with scaling does not help.

![Screenshot from 2023-12-16 21:45:33](https://github.com/alicevision/Meshroom/assets/10870125/e6890a47-efae-41ee-bb90-86ee53a533b3)

**Reply by natowi** [reply]
You need to look for linux specific guides on the topic: https://wiki.archlinux.org/title/HiDPI#Qt_5

There are parameters like QT_SCALE_FACTOR, QT_FONT_DPI and QT_SCALE_FACTOR_ROUNDING you could test.

**Comment by roydenyates** [comment]
see this [bug](https://github.com/alicevision/Meshroom/issues/2303).
